tools/acorn-optimizer.mjs

```diff
@@ -289,7 +289,7 @@ function JSDCE(ast, aggressive) {
     }
     function cleanUp(ast, names) {
       recursiveWalk(ast, {
-        VariableDeclaration(node, c) {
```

Comment:
is this always good to do? if the caller provides the extra param, not having it might end up going through a slow path. i mean that matching the caller seems optimal, or at least that was the case years ago.

Reply:
OK, revert this part